common/src/main/java/net/adoptopenjdk/icedteaweb/lockingfile/LockableFile.java

```diff
@@ -88,12 +91,17 @@ public static LockableFile getInstance(final File file) {
     private LockableFile(final File file) {
         this.file = file;
         try {
-            //just try to create
-            this.file.createNewFile();
+            if (! file.exists()) {
+                if (! file.createNewFile()) {
+                    throw new IOException("could not create file " + file);
+                }
+            } else if (! file.isFile()) {
+                logger.error("lockable file {} is not a file but something else (maybe a directory)", file);
+            }
         } catch (final Exception ex) {
-            //intentionally silent
+            logger.error("Exception while creating lockable file", ex);
```

Comment:
In general I ask myself if throwing exceptions would not make more sense. But this would maybe change the runtime behavior...

Reply:
Yes this is the main reason, try to alter the current behavior as little as possible